ci/rpm8/build-dep.sh

```diff
@@ -23,6 +23,7 @@
 
 # FIXME: default yum repositories has been changed to vault due to CentOS 8 EOL on February 1st, 2022
 # This is just a temporal solution so the build doesn't break while we find a new distro to use
+# FIXME: python2 required by an installagion script in GMock. Sad but true :(
```

Comment:
```suggestion
# FIXME: python2 required by an installation script in GMock. Sad but true :(
```

Reply:
Fixed in [f4765cc](https://github.com/telefonicaid/fiware-orion/pull/4059/commits/f4765cc26e4063609484891074063a7fdf51cd74)